internal/ykclient/client.go

```diff
@@ -36,9 +33,18 @@ var (
 	// nodeEndPt  = func(partitionName, nodeID string) string {
 	// 	return fmt.Sprintf("/ws/v1/partition/%s/node/%s", partitionName, nodeID)
 	// }
-	nodeUtilEndPt     = "/ws/v1/scheduler/node-utilizations"
-	applicationsEndPt = func(partitionName, queueName string) string {
-		return fmt.Sprintf("/ws/v1/partition/%s/queue/%s.%s/applications", partitionName, queueName, partitionName)
+	nodeUtilEndPt           = "/ws/v1/scheduler/node-utilizations"
+	applicationsByPartEndPt = func(partitionName string) string {
+		return fmt.Sprintf("/ws/v1/partition/%s/applications/active", partitionName)
```

Comment:
for the `/active` part https://yunikorn.apache.org/docs/next/api/scheduler/#partition-applications